website/pages/docs/configuration/audit.mdx

```diff
@@ -54,7 +57,7 @@ in order for HTTP requests to successfully complete.
 - `sink` <code>([sink](#sink-stanza): default)</code> - Configures a sink
   for audit logs to be sent to.
 
-- `filter` <code>([filter](#filter-stanza): nil)</code> - Configures a filter
+- `filter` <code>(<array>[filter](#filter-stanza): nil)</code> - Configures a filter
```

Comment:
Looks like the array syntax is a bit different from others.  Also, should this default empty list in docs?

```suggestion
- `filter` <code>(array<[filter](#filter-stanza)>: [])</code> - Configures a filter
```